ci/jenkins/test-vmc.sh

```diff
@@ -202,6 +228,10 @@ function setup_cluster() {
         sed -i "s|${kv[0]}|${kv[1]}|g" ${GIT_CHECKOUT_DIR}/jenkins/out/cluster.yaml
     done < "$cluster_defaults"
 
+    #only for test
+    cp ${GIT_CHECKOUT_DIR}/jenkins/out/cluster.yaml /tmp/cluster.yaml
+    cp ${GIT_CHECKOUT_DIR}/jenkins/out/cluster.yaml /tmp/namespace.yaml
+
     echo '=== Create a cluster in management cluster ==='
     kubectl apply -f "${GIT_CHECKOUT_DIR}/jenkins/out/namespace.yaml"
     kubectl apply -f "${GIT_CHECKOUT_DIR}/jenkins/out/cluster.yaml"
```

Comment:
In line 239, 240, 261, 264, it only waits for 10 mins. Now that we use NFS, I suggest that we change to 20 minutes timeout.